symfony/framework-bundle/3.3/src/Kernel.php

```diff
@@ -35,6 +36,27 @@ public function registerBundles()
         }
     }
 
+    public static function bootstrapEnvironment(string $env = null)
+    {
+        $env = $env ?? $_SERVER['APP_ENV'] ?? null;
```

Comment:
If we decide to include `$env ?? $_SERVER['APP_ENV'] ?? $_ENV['APP_ENV'] ?? null` we can remove the tenerary on phpunit bootsrap.

Reply:
I think we should do this. But, read `$_ENV['APP_ENV']` first - it should take priority. This is consistent with how env vars are read in the container: 

https://github.com/symfony/symfony/blob/4f4c1725f0fcc2bb858ef0e2b5b892c5dac253a8/src/Symfony/Component/DependencyInjection/EnvVarProcessor.php#L71-L74

Reply:
@weaverryan according to with what you just reviewed later:

> So, for me, this $env argument is more of an optional, "override" - for a case where you want to use this $env even IF $_SERVER['APP_ENV'] is already set. So, it should be optional and should not be passed, except from the console.

This is the whole purpose of $env, so i think it should be read first, and fallback to the APP_ENV if we are not overriding it.